src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -25,12 +28,24 @@ class TimezoneType extends AbstractType implements ChoiceLoaderInterface
      * The choices are generated from the ICU function \DateTimeZone::listIdentifiers().
      *
      * @var ArrayChoiceList
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
      */
     private $choiceList;
 
     /**
      * {@inheritdoc}
      */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        if ('datetimezone' === $options['input']) {
```

Comment:
How about we name the option value `DateTimeZone` so people can use `['input' => \DateTimeZone::class]`? This avoids typos and makes things more explicit as the value directly maps to a class.